app/controllers/videocall_controller.rb

```diff
@@ -0,0 +1,7 @@
+class VideocallController < ApplicationController
```

Comment:
Why do we need this ?

Reply:
We do not need the index action, removed. However we do need the controller for routing.

Reply:
We tried to remove it but got uninitialized controller error.